mappings/net/minecraft/entity/ai/goal/MoveToVillageCenterGoal.mapping

```diff
@@ -0,0 +1,6 @@
+CLASS akj net/minecraft/entity/ai/goal/MoveToVillageCenterGoal
+	FIELD a owner Laib;
+	FIELD b searchRange I
+	FIELD c reciprocalChance I
```

Comment:
It's the reciprocal chance of doing what?

Reply:
Of being triggered for an update each tick.